.github/workflows/rust.yml

```diff
@@ -1,4 +1,5 @@
-name: Cargo fmt
+# Rust specific tests
```

Comment:
I thought we would keep "# Rust specific tests like formatting, clippy" and just remove the "etc.".

Reply:
Let's go with this, no risk of outdated comments this way...